scalafix-core/shared/src/main/scala/scalafix/internal/rule/ExplicitResultTypes.scala

```diff
@@ -74,9 +75,16 @@ case class ExplicitResultTypes(
     def defnType(defn: Defn): Option[(Type, Patch)] =
       for {
         name <- defnName(defn)
-        symbol <- name.symbol
-        typ <- symbol.resultType
-      } yield TypeSyntax.prettify(typ, ctx, config.unsafeShortenNames)
+        denot <- name.denotation
+        tpe <- denot.tpeInternal
+        // Skip existential types for now since they cause problems.
+        if !tpe.tag.isExistentialType
+        result <- TypeSyntax.prettify(
+          tpe,
+          ctx,
+          config.unsafeShortenNames,
```

Comment:
How is this config passed? https://scalacenter.github.io/scalafix/docs/users/configuration shows how to add rules to the scalafix config, but does not show if it is possible to pass rule specific config.

Reply:
It's configuration for ExplicitResultTypes documented here https://scalacenter.github.io/scalafix/docs/rules/ExplicitResultTypes

Reply:
I wasn’t able to put that together from the docs for some reason but I get it now, thanks.